src/engine/shared/config_weapons.h

```diff
@@ -28,6 +28,8 @@ MACRO_CONFIG_INT(InfShotgunBlowback, inf_shotgun_blowback, 2, 0, 15, CFGFLAG_SER
 MACRO_CONFIG_INT(InfMedicShotgunBlowback, inf_medic_shotgun_blowback, 10, 0, 15, CFGFLAG_SERVER, "medic shotgun blowback")
 MACRO_CONFIG_INT(InfBioLaserDamage, inf_bio_laser_damage, 10, 0, 20, CFGFLAG_SERVER, "Damage in hearts")
 MACRO_CONFIG_INT(InfBioLaserEnergy, inf_bio_laser_energy, 400, 0, 1000, CFGFLAG_SERVER, "Energy of biologist laser")
+MACRO_CONFIG_INT(InfSniperLaserBaseDmg, inf_sniper_laser_base_dmg, 10, 0, 20, CFGFLAG_SERVER, "Damage in hearts")
```

Comment:
this is not the equivalent of what was before

Reply:
Here we can see what was before:
https://github.com/yavl/teeworlds-infclassR/blob/master/src/game/server/entities/character.cpp#L1363
If he is in locked position he will do 30 damage, if not he will do 10 - 13 randomly

Reply:
Is there a reason why classes do not deal predictable damage? I thought this is our standard.

Reply:
I have added it.